.github/workflows/build_and_test.yml

```diff
@@ -129,6 +129,7 @@ jobs:
     name: "Build docs"
     uses: ./.github/workflows/_docs.yml
     needs: build-torch-xla
+    if: github.event_name == 'push'
```

Comment:
Interesting find... I guess previously we were pushing docs even from pending PR branches? Seems risky

Reply:
This used to be inside the `_docs.yml` but I moved it here so we can skip the workflow altogether for non-push action because we are uploading docs on push commit anyway. Previously, all the steps up to the Deploy step from `_docs.yml` would be run.